lexos/helpers/general_functions.py

```diff
@@ -230,17 +232,49 @@ def _try_decode_bytes_(raw_bytes: bytes) -> str:
     return decoded_string
 
 
-def decode_bytes(raw_bytes: bytes) -> str:
-    """Decode the raw byte into a string
+def decode_bytes(raw_bytes: Union[bytes, str]) -> str:
```

Comment:
Interesting, why do you think this function needs to handle `str` given that the function name is called `decode_byte`